python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -233,6 +244,7 @@ def solid_handle(self) -> NodeHandle:
         return self._step_execution_context.solid_handle
 
     @property
+    @public
     def op_handle(self) -> NodeHandle:
         """NodeHandle: The current op's handle.
```

Comment:
why do we want to make this public?

Reply:
Hmm I think we do not. Thanks for catching